File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
import sys

n, m = map(int, sys.stdin.readline().split())

matrix_a = []
for _ in range(n):
    matrix_a.append(list(map(int, sys.stdin.readline().split())))

matrix_b = []
for _ in range(n):
    matrix_b.append(list(map(int, sys.stdin.readline().split())))

for i in range(n):
    for j in range(m):
        print(matrix_a[i][j] + matrix_b[i][j], end=" ")
    print()

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');

const [N, M] = input[0].split(' ').map(Number);

const matrixA = input.slice(1, N + 1).map(line => line.split(' ').map(Number));
const matrixB = input.slice(N + 1).map(line => line.split(' ').map(Number));

for (let i = 0; i < N; i++) {
    let row = [];
    for (let j = 0; j < M; j++) {
        row.push(matrixA[i][j] + matrixB[i][j]);
    }
    console.log(row.join(' '));
}